opentelemetry-sdk/src/metrics/meter.rs

```diff
@@ -394,6 +394,23 @@ impl SdkMeter {
             return Histogram::new(Arc::new(NoopSyncInstrument::new()));
         }
 
+        if let Some(ref boundaries) = builder.boundaries {
```

Comment:
Let's remove the validation checks from the `Histogram` struct now that we would be checking it during instrument creation.

https://github.com/open-telemetry/opentelemetry-rust/blob/8e6b479684d550fb70b570d059080488b893aaf4/opentelemetry-sdk/src/metrics/internal/histogram.rs#L78-L79

Reply:
The view route is unvalidated still I think. Something we need to comeback to and fix.

Reply:
Ah yes.